benchmarks/src/main/scala/zio/zmx/package.scala

```diff
@@ -0,0 +1,30 @@
+package zio
+
+package object zmx {
+  def spawn(i: Int): ZIO[Any, Nothing, Fiber.Runtime[Nothing, Unit]] =
+    if (i <= 0) {
+      ZIO.never.fork
```

Comment:
```suggestion
      ZIO.unit
```

Reply:
@adamgfraser won't the fibers terminate nearly immediately (as they're joined right after spawned), making our graph possibly not big enough?

Reply:
Well right now we're benchmarking the overhead of writing to this concurrent set so that cost will get incurred every time we fork or join a fiber. We're not doing anything with the concurrent set but you're right if we were benchmarking operations on the concurrent set it would be important to make sure it was of the desired size.

Reply:
sure! updated in https://github.com/zio/zio-zmx/pull/135/commits/cab61eb7f96d55c11127a95f3dc32f732ffa020f